pkgs/applications/video/gnome-mpv/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchurl
-, intltool, pkgconfig, wrapGAppsHook
+{ stdenv, fetchFromGitHub, meson, ninja
+, gettext, pkgconfig, desktop_file_utils, wrapGAppsHook
 , appstream-glib, epoxy, glib, gtk3, mpv
 }:
 
 stdenv.mkDerivation rec {
   name = "gnome-mpv-${version}";
-  version = "0.9";
+  version = "0.13";
 
-  src = fetchurl {
-    sha256 = "06pgxl6f3kkgxv8nlmyl7gy3pg55sqf8vgr8m6426mlpm4p3qdn0";
-    url = "https://github.com/gnome-mpv/gnome-mpv/releases/download/v${version}/${name}.tar.xz";
+  src = fetchFromGitHub {
+    owner = "gnome-mpv";
+    repo = "gnome-mpv";
+    rev = "v${version}";
+    sha256 = "0ismh56k0alzplhpimzb2wlhj0bgqiw4hsqrz917vdgly1959ghg";
   };
```

Comment:
For the record, the reason not to use the official release is that it misses some stuff present in the corresponding tag: https://github.com/gnome-mpv/gnome-mpv/releases/tag/v0.13

Reply:
Fixed by using the daughter commit, that fixes the appdata – the patch does not make sense.

Reply:
Fine. I also meant that although in general GitHub *releases* (rather than *archives*) should not be replaced with `fetchFromGitHub`, here it is justified.

Reply:
I do not think meson distinguishes between a source and a release archive.

IMO, the distinction is confusing anyway – for example, I was just wondering how does [this expression](https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/gnome-3/misc/libgames-support/default.nix) build without `vala` dependency, when it is [pure Vala library](https://github.com/GNOME/libgnome-games-support).